### ISSUE TITLE
Error with time_filter

### ISSUE THREAD

**Issue by pssguy** [business-science/tibbletime#6]
Just working through your blog post with the CRAN version 0.0.1

```
library(tibbletime)
data(FB)
FB <- as_tbl_time(FB, index = date)
class(FB)  #[1] "tbl_time"   "tbl_df"     "tbl"        "data.frame"
FB %>% 
  time_filter(2013-03-15 ~ 2015-12-31)
Incompatible methods ("Ops.Date", "Ops.POSIXt") for ">="Incompatible methods ("Ops.Date", "Ops.POSIXt") for "<="
```

Installed dev version (0.0.1.9000)  

```
library(tibbletime)
data(FB)
FB 
#Error in is_null(n) : object 'rlang_is_null' not found
```

**Comment by pssguy** [comment]
I had an issue with rlang which once fixed meant that v 0.0.1.9000 now works

**Comment by DavisVaughan** [comment]
@pssguy, yes I found the first problem with `time_filter` immediately after sending to CRAN and it has been fixed in the dev version. It's an odd problem that you sometimes can't compare a:
 `Date_class_object > POSIXt_class_object`

Glad you got the rlang issue fixed.
